osf/migrations/0137_add_fm_record_to_osfstorage_files.py

```diff
@@ -1,32 +1,58 @@
 # -*- coding: utf-8 -*-
 # Generated by Django 1.11.13 on 2018-09-26 15:18
 from __future__ import unicode_literals
+from math import ceil
+import logging
 
 from django.db import migrations, connection
 
+logger = logging.getLogger(__name__)
+
+increment = 100000
 
 def remove_records_from_files(state, schema):
     FileMetadataRecord = state.get_model('osf', 'filemetadatarecord')
     FileMetadataRecord.objects.all().delete()
 
+# Batching adapted from strategy in website/search_migration/migrate.py
 def add_records_to_files_sql(state, schema):
     FileMetadataSchema = state.get_model('osf', 'filemetadataschema')
     datacite_schema_id = FileMetadataSchema.objects.filter(_id='datacite').values_list('id', flat=True)[0]
+    OsfStorageFile = state.get_model('osf', 'osfstoragefile')
+    max_fid = OsfStorageFile.objects.last().id
+
     sql = """
-    INSERT INTO osf_filemetadatarecord (created, modified, _id, metadata, file_id, schema_id)
-    SELECT  NOW(), NOW(), generate_object_id(), '{}', OSF_FILE.id, %d
-        FROM osf_basefilenode OSF_FILE
-            LEFT OUTER JOIN osf_filemetadatarecord ON
-                (OSF_FILE.id = osf_filemetadatarecord.file_id)
```

Comment:
Removed this JOIN - is this needed? I'm assuming osf_filemetadatarecord table is not populated.

Reply:
Seems like a valid assumption, as this table was just created in the previous migration.